.env.prod

```diff
@@ -0,0 +1,10 @@
+FLASK_APP=app.py
+FLASK_DEBUG=1
+DATABASE_URL=postgresql://hello_flask:hello_flask@db:5432/users_hash
+SQL_HOST=db
+SQL_PORT=5432
+DATABASE=postgres
+SQLALCHEMY_ECHO=True
+SQLALCHEMY_TRACK_MODICATIONS=False
+SECRET_KEY=2333
```

Comment:
Have no idea what this was but I saww it in your app.py so I added them as OS variables